src/components/Selector.tsx

```
@@ -0,0 +1,43 @@
+      </Text>
+
+      {values.map((v) => (
+        <Button
+          radius={0}
+          variant={selected === v ? "filled" : "outline"}
+          onClick={() => onSelect?.(v)}
+          style={{
+            padding: "8px",
+            margin: 0,
+          }}
+        >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)